From 54275ddcc2284e5c244747db826bfc91a65a67f8 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@apache.org>
Date: Thu, 21 Aug 2014 18:58:43 +0000
Subject: [PATCH 199/791] HADOOP-10431. Change visibility of KeyStore.Options
 getter methods to public. (tucu)

Conflicts:
	hadoop-common-project/hadoop-common/CHANGES.txt

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-2@1619516 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit ab7ed6d9b4572f3ac7024efea07c0ffa175922d6)

Conflicts:
	hadoop-common-project/hadoop-common/CHANGES.txt
---
 .../org/apache/hadoop/crypto/key/KeyProvider.java  |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/KeyProvider.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/KeyProvider.java
index b2177da..a8a303c 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/KeyProvider.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/KeyProvider.java
@@ -260,15 +260,15 @@ public Options setDescription(String description) {
       return this;
     }
 
-    protected String getCipher() {
+    public String getCipher() {
       return cipher;
     }
 
-    protected int getBitLength() {
+    public int getBitLength() {
       return bitLength;
     }
 
-    protected String getDescription() {
+    public String getDescription() {
       return description;
     }
   }
-- 
1.7.9.5

